auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -110,8 +109,11 @@
         </template>
         <template v-slot:no-data>
           <span v-if="loading">Loading...</span>
-          <span v-else>Add an existing company, cooperative or society to manage it or<br> add a Name Request to
-            complete your incorporation or registration.</span>
+          <div v-else>
+            Add an existing company, cooperative or society to manage it or
+            <br>
+            add a Name Request to complete your incorporation or registration.
+          </div>
```

Comment:
There's really little difference here, but it's better to not pretend this is an inline element when it's actually a block-level element.